Objective-C/Tests/LogTest.m

```diff
@@ -137,7 +178,7 @@ - (void) testPlainTextLoggingLevels {
     for (NSString* file in files) {
         NSString* log = [path stringByAppendingPathComponent: file];
         NSString* content = [NSString stringWithContentsOfFile: log
-                                                      encoding: NSUTF8StringEncoding
+                                                      encoding: NSASCIIStringEncoding
```

Comment:
this is to potential fix for the Jenkins failure. 

Error at :0: -[CouchbaseLiteSwiftTests.LogTest testPlainTextLoggingLevels] : failed: caught error: The file “cbl_warning_1547058710.cbllog” couldn’t be opened using text encoding Unicode (UTF-8).

Reply:
wrong!